store/src/java/com/zimbra/cs/util/AccountUtil.java

```diff
@@ -928,7 +928,33 @@ public static boolean isZulipChatEnabled(Account account) {
         }
         return isEnabled;
     }
-    
+
+    public static boolean isDocumentEditingEnabled(Account account) {
+        boolean isEnabled = false;
+        if (account == null) {
+            return isEnabled;
+        }
+        isEnabled = account.isFeatureDocumentEditingEnabled();
+        if (isEnabled) {
+            Provisioning prov = Provisioning.getInstance();
+            try {
+                Server server = prov.getServer(account);
+                if (server == null) {
+                    ZimbraLog.account.warn("no server associated with acccount " + account.getName());
+                    isEnabled = false;
+                } else {
+                    String value = server.getDocumentServerHost();
+                    if (!StringUtil.isNullOrEmpty(value)) {
```

Comment:
Here we just want to make sure, the server is associated, right?

Reply:
Yes @RakeshAMore